tests/lax_scipy_test.py

```diff
@@ -132,7 +138,15 @@ def testScipySpecialFun(self, scipy_op, lax_op, rng_factory, shapes, dtypes,
     self._CompileAndCheck(lax_op, args_maker, rtol=1e-5)
```

Comment:
If you change this to `rtol=1E-4`, I think that will satisfy all the tests. I believe the issue is that the compiled version re-orders some of the floating point ops, which leads to larger differences between the compiled and non-compiled versions.

Reply:
Thank you! I have changed it to `1e-4`. :)